README.md

```diff
@@ -1457,6 +1457,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Thisaru Dilshan](https://github.com/ThisaruD)
   - [Tibebe Solomon](https://github.com/sapencio)
   - [Tikeshwar Sahu](https://github.com/Tiku24)
+  - [Tierra Barrow](https://github.com/tcodes27)
```

Comment:
Please put your name before the name "Tikeshwar Sahu". It should be in the alphabetical order.